### PR TITLE
Switch wip to 4 5 0

### PR DESCRIPTION
**Pull request by corest**
WIP versions need namespace labeling to work with network policies

### CODE REVIEW

File: service/controller/v8/version_bundle.go [hunk 1]
```diff
@@ -27,6 +27,11 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Update to 3.3.13. More info here: https://github.com/etcd-io/etcd/blob/master/CHANGELOG-3.3.md#v3313-2019-05-02",
 				Kind:        versionbundle.KindChanged,
 			},
+			{
+				Component:   "ignition",
+				Description: "Add name label for default and kube-system namespaces.",
+				Kind:        versionbundle.KindAdded,
+			},
```
Comment (kopiczko) [inline service/controller/v8/version_bundle.go]: Isn't v8 already release?

Comment (corest) [reply service/controller/v8/version_bundle.go]: `v8` is `2.4.0` and it is WIP

Comment (kopiczko) [reply service/controller/v8/version_bundle.go]: So why do we have v9?

Comment (corest) [reply service/controller/v8/version_bundle.go]: That's a question to @mazzy89 https://github.com/giantswarm/azure-operator/pull/525

**Pull request merged**